recipe/migrations/gdal32.yaml

```diff
@@ -1,7 +1,9 @@
 __migrator:
   build_number: 1
   kind: version
-  migration_number: 1
+  migration_number: 2
 gdal:
 - '3.2'
-migrator_ts: 1604390922.414637
+libgdal:
+- '3.2'
+migrator_ts: 1614669979
```

Comment:
@xhochy, is this what you had in mind?

Reply:
Keep everything the same except adding `libgdal` here. Once merged and build and CDN-synced (i.e. 40min later), add a bot-rerun label to the failing PRs.

Increasing the migration number would restart the whole migration otherwise.

Reply:
@xhochy, the problem is not failing PRs, it's whole feedstock that the bot skipped because they only depend on `libgdal`, not `gdal`.  I don't think your approach will work.

Reply:
@ocefpaf, @CJ-Wright or @beckermr, do you care to weigh in on what should happen?  I'm happy to just add `libgdal` without changing `migrator_ts` or `migration_number` if that will do the trick.

Reply:
The migratior ts shouldn't matter, that is used for ordering the application of the pins (so there is an order of operations in case one interacts with the other). I don't think we need to bump the migration number since we don't need to start from scratch. Adding libgdal should do the trick, the bot understand that libgdal is also being migrated and add feedstocks accordingly

Reply:
it matters to conda-smithy, so don't change it.

Reply:
@isuruf, could you clarify what you would like to see happen?  Just adding `libgdal` and no other change?

Reply:
Yes, see @CJ-Wright's suggestions.